docs/_docs/collections.md

```diff
@@ -44,7 +44,10 @@ collections:
 Create a corresponding folder (e.g. `<source>/_staff_members`) and add
 documents. Front matter is processed if the front matter exists, and everything
 after the front matter is pushed into the document's `content` attribute. If no front
-matter is provided, Jekyll will not generate the file in your collection.
+matter is provided, Jekyll will consider it to be a [static file](https://jekyllrb.com/docs/static-files/)
```

Comment:
Please consider using host-agnostic links so that they can be traversed locally:
 ```suggestion
matter is provided, Jekyll will consider it to be a [static file](/docs/static-files/)
```